Source/ACE/Entity/Landblock.cs

```diff
@@ -467,6 +472,90 @@ private void HandleGameAction(QueuedGameAction action, Player player)
+                                        aPlayer.RemoveFromInventory(inventoryId);
+                                    }
+                                }
+                                
+                                // We are droping the item - let's keep track of change in burden
+
+                                if ((aPlayer != null) && (inventoryItem != null))
+                                { 
+                                    aPlayer.GameData.Burden -= inventoryItem.GameData.Burden;
+
+                                    // OK, now let's tell the world and our client what we have done.
+                                    var targetContainer = new ObjectGuid(0);
+                                    aPlayer.Session.Network.EnqueueSend(
+                                        new GameMessagePrivateUpdatePropertyInt(aPlayer.Session,
+                                            PropertyInt.EncumbVal,
+                                            (uint)aPlayer.Session.Player.GameData.Burden));
+                                    var motion = new GeneralMotion(MotionStance.Standing);
+                                    motion.MovementData.ForwardCommand = 24;
+                                    aPlayer.Session.Network.EnqueueSend(new GameMessageUpdateMotion(aPlayer, aPlayer.Session, motion));
+
+                                    // Set Container id to 0 - you are free
+                                    aPlayer.Session.Network.EnqueueSend(
+                                        new GameMessageUpdateInstanceId(inventoryId, targetContainer));
+
+                                    motion = new GeneralMotion(MotionStance.Standing);
+                                    aPlayer.Session.Network.EnqueueSend(new GameMessageUpdateMotion(aPlayer, aPlayer.Session, motion));
+
+                                    // Ok, we can do the last 3 steps together.   Not sure if it is better to break this stuff our for clarity
+                                    // Put the darn thing in 3d space
+                                    // Make the thud sound
+                                    // Send the container update again.   I have no idea why, but that is what they did in live.
+                                    aPlayer.Session.Network.EnqueueSend(
+                                        new GameMessagePutObjectIn3d(aPlayer.Session, aPlayer, inventoryId),
+                                        new GameMessageSound(aPlayer.Guid, Sound.DropItem, (float)1.0),
+                                        new GameMessageUpdateInstanceId(inventoryId, targetContainer));
+                                    // Set the flags and determine a position.
+
+                                    // TODO: I need to look at the PCAPS to see the delta in position from the dropper and the item dropped.   Temp position.
+                                    
+                                    inventoryItem.PositionFlag = UpdatePositionFlag.Contact

```

Comment:
modification of the inventoryItem's position/whatever data should stay here in the landblock, yes.